### DISCUSSION TITLE
Huge windows bar after update 0.72 --> 0.74

### DISCUSSION THREAD

**Discussion by tomaskcz** [zim-desktop-wiki/zim-desktop-wiki#1753] [Q&A] [answered]
![Screenshot_2021-11-19_14-04-34](https://user-images.githubusercontent.com/90001542/142628111-1ca011d6-a817-411f-a04c-168b026d3b55.png)

Zim now having huge window bar taking lots of screen pixels - is there any way to get back classic window bar accorging to the linux desktop environment settings ?

Using Xfce latest version.

**Comment by dehesselle** [comment, marked as answer]
I think you're talking about the CSD which can be disabled, see item 2 in [this comment](https://github.com/zim-desktop-wiki/zim-desktop-wiki/issues/1582#issuecomment-936100830) on how to disable.

**Comment by tomaskcz** [comment]
Unfortunately no, enabling this plugins adds an extra toolbar, but has no effect on the window bar itself. I had this plugin disabled.

**Reply by dehesselle** [reply]
I don't think you're looking in the right place. Please read again item 2 in the link I posted; this is not about the tool bar, there is an option to disable CSD ("show controls in windows decoration"). The screenshot you posted clearly shows that CSD is enabled, otherwise that home button wouldn't be there in the window bar.

This is what CSD on/off looks like on my Gnome desktop (Adwaita theme):
![zim_csd](https://user-images.githubusercontent.com/11375494/142694914-127a54c9-5e2c-4168-b7c6-534baa73cfcf.png)

**Reply by tomaskcz** [reply]
ok, I managed to get it back.
The steps I needed to save max. vert. pixel size:
- activate Tool bar plugin (disabled by default)
- untick all options there
- restart the app (I missed the restart before, that's why it had no effect)

Thank you

**Reply by dehesselle** [reply]
Great! Can you mark this as "answered" please?
